service/collector/orphan_configmap.go

```diff
@@ -17,7 +17,7 @@ import (
 
 var (
 	orphanConfigMapDesc *prometheus.Desc = prometheus.NewDesc(
-		prometheus.BuildFQName(Namespace, "configmap", "orphan"),
+		prometheus.BuildFQName(namespace, "configmap", "orphan"),
```

Comment:
For my interest does this mean we also need to change g8s-prometheus?

Reply:
No, I changed the name of`Namespace` variable into `namespace`, so no need to change anything from g8s-prometheus. Sorry for the confusion.

Reply:
No problem and I could have got that. Sorry it took so long to get back to this.